src/radiology-tabs/common/listOrderDetails.scss

```
@@ -0,0 +1,57 @@
+@use '../../../node_modules/@carbon/styles/scss/spacing';
+@use '../../../node_modules/@carbon/styles/scss/type/index';
+@import "../../../node_modules/@openmrs/esm-styleguide/src/vars";
+
+.orderTile {
+  width: 100%;
+  margin: 2px 0 8px;
+  padding: 0 8px 0 8px;
+  background-color: #fff;
+  border-left: 4px solid var(--brand-03);
+  color: $text-02;
+  margin-bottom: 1rem !important;
```

```suggestion
  margin-bottom: 1rem !important;
```
if possible using `!important` should be the last options